### PR TITLE
Add sampling documentation

### PR DESCRIPTION
**Pull request by black-adder**
Signed-off-by: Won Jun Jang <wjang@uber.com>

### CODE REVIEW

**Comment by lucperkins** [comment]
@black-adder I'm happy to make the necessary changes to bring the sampling doc in line with the new Hugo-based setup. If you give me write access to your fork I can just push those changes directly to this PR.

File: docs/sampling.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  ],
+  "default_strategy": {
+    "type": "probabilistic",
+    "param": 0.5
+  }
+}
+```
+
+`service_strategies` defines service specific sampling strategies. There are 2 types of strategies possible: `probabilistic` and `ratelimiting` which are described [above](#client-sampling-configuration). `default_strategy` defines the catch-all sampling strategy that is propagated if the service is not included as part of `service_strategies`.
```
Comment (yurishkuro) [inline docs/sampling.md]: Q: is the per-endpoint configuration possible via this file?

Comment (black-adder) [reply docs/sampling.md]: not per-endpoint, only per service

File: content/docs/sampling.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: Sampling
+rank: 8
```
Comment (yurishkuro) [inline content/docs/sampling.md]: I think Sampling should be next to Client Libraries.

Comment (lucperkins) [reply content/docs/sampling.md]: Okay I'll re-adjust the ranking.

**Comment by yurishkuro** [comment]
Q: do we support nested hierarchy for the main menu? Sampling is kind of in between top-level and a subsection of Client Libraries (but for now top level is fine)

**Comment by lucperkins** [comment]
@yurishkuro That's not supported yet, no. I can switch over to a system like that at any point. Just give me a heads up.

**Pull request merged**